### DISCUSSION TITLE
Hacktoberfest

### DISCUSSION THREAD

**Discussion by mrsOwlex** [riok/mapperly#784] [General]
👋 Hello everyone!

🎉 We're gearing up for a fantastic Hacktoberfest event at my company this year! As part of our preparations, we're scouting for great open-source projects to contribute to, and we'd like to suggest that repositories participating in Hacktoberfest consider adding the "Hacktoberfest" label to make it easier for contributors to find relevant issues and pull requests. 🏷️

By doing this, you not only encourage us but also many others in the open-source community to get involved in your project. Rest assured, we'll adhere to your project's guidelines and contribute meaningfully.

If your project has specific Hacktoberfest participation guidelines, kindly let us know, and we'll ensure our contributions align with your expectations.

Thanks for considering our request, and we look forward to contributing to your project during this exciting event! 😄

**Comment by latonz** [comment]
Thanks for bringing this up, I have added the `hacktoberfest` topic. Looking forward to your contributions 😊
